sdk/resourcemanager/Azure.ResourceManager.Core/src/PropertyWrapper.cs

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using Azure.ResourceManager.Resources;
+using System.Reflection;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Threading.Tasks;
+
+namespace Azure.ResourceManager.Core
+{
+    internal class PropertyWrapper
+    {
+        internal PropertyInfo Info { get; }
+
+        internal object PropertyObject { get; }
+
+        internal PropertyWrapper(PropertyInfo info, object propObject)
```

Comment:
Instead of exposing the PropertyInfo object, just expose a GetValue() and SetValue(string) which use the propertyInfo internally

Reply:
If we do this, don't we have to in BuildApi table do:

new PropertyWrapper()

PropertyWrapper.PropertyObject = obj;
PropertyWrapper.Info = info;

What is the suggestion here?